src/app/(admin)/new_dashboard/guides/hooks/useSearchGuides.ts

```diff
@@ -12,7 +14,16 @@ export const useSearchGuides = (initialGuides: Guide[]) => {
     const matchesTag =
       selectedTag === "All" || guide.categories.includes(selectedTag);
     return matchesQuery && matchesTag;
-  });
+  })
+    .sort((a, b) => {
```

Comment:
extract this whole sorting function

Reply:
done to `sortByCriteria`  from "./sortGuides";